docs/source/getting-started/installation.rst

```diff
@@ -29,22 +29,27 @@ Download and run the installer_, which will install DAML and set up your PATH.
 Mac and Linux
 =============
 
-To install the SDK on Mac or Linux:
-
-1. In a terminal, run:
+To install the SDK on Mac or Linux open a terminal and run:
 
    .. code-block:: shell
 
       curl -sSL https://get.daml.com/ | sh
 
-2. Add ``~/.daml/bin`` to your PATH. You can find the Mac OS and Linux instructions :doc:`here <path-variables>`.
+The installer will setup the PATH variable for you. In order for it to take effect, you will have to
+logout and login again.
```

Comment:
That's a tricky one. IIRC, "login" and "logout" are nouns, "log in" and "log out" the corresponding verbs.
```suggestion
log out and log in again.
```